include/readdy/kernel/singlecpu/model/SCPUParticleData.h

```diff
@@ -152,39 +177,39 @@ class SCPUParticleData {
         }
     }
 
-    iterator begin() {
+    Iterator begin() {
```

Comment:
in terms of naming consistency to the stl id actually go for `iterator` and `const_iterator` in this case